pyproject.toml

```diff
@@ -33,6 +34,15 @@ build-book-dev.shell = """
   jupyter-book build src/almanac && \
   python -m webbrowser -t "file://$PWD/src/almanac/_build/html/index.html"
 """
+# runs vale relative to this project through pre-commit
+# to trigger warnings as non-zero return when there is
+# more than one line of output from vale.
+vale-checks.shell = """
+output=$(poetry run vale ./src/almanac) && \
+if [ $(echo $output | wc -l) -gt 1 ]; then \
+echo $output && exit 1; \
+fi
+"""
```

Comment:
I'd noticed when I tried to run `pre-commit run --all-files` that this command  was failing with the message `[loadStyles] Runtime error style 'proselint' does not exist on StylesPath`. I took me a while to figure out, by reading the GH workflow, that I should have run `poetry run vale sync` to pull the vale styles(?). Perhaps adding it here, or at least instructing the user to run it, would be helpful?

Reply:
Thanks for this, great point! I'll add another part to this shell command to sync before running Vale.